microsoft/botframework-solutions
# [BotBuilder Libs] Test structure refactor

## Description

Update the following libs to follow the test structure of the rest of the solutions
- `botbuilder-solutions` 
- `botbuilder-skills` 

## Testing Steps
1. Go to `botframework-solutions\lib\typescript`
1. Check the new structure for the tests in `botbuilder-skills` and `botbuilder-solutions`
1. Run `rush update` and `rush rebuild`
1. Run `rush test` and check that all the tests are passing successfully

## Checklist
